react/components/UI/ModalDialog/index.js

```diff
@@ -8,7 +8,7 @@ const Dialog = styled.div`
   width: 90%;
   height: 90%;
   max-width: 40em;
-  max-height: 60em;
+  max-height: calc(100vh - 7em);
```

Comment:
This is gonna look odd on super tall screens. If this is a workaround for Safari there's a better way (I don't know off hand)

Reply:
Fine for now though. But I'd leave a note about the underlying issue: Safari flexbox blah blah